kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala

```diff
@@ -88,12 +100,16 @@ private[v1] class SessionsResource extends ApiRequestContext {
   @DELETE
   @Path("{sessionHandle}")
   def closeSession(@PathParam("sessionHandle") sessionHandleStr: String): Response = {
+    val sessionHandle = getSessionHandle(sessionHandleStr)
+    backendService.closeSession(sessionHandle)
+    Response.ok().build()
+  }
+
+  def getSessionHandle(sessionHandleStr: String): SessionHandle = {
     val splitSessionHandle = sessionHandleStr.split("\\|")
     val handleIdentifier = new HandleIdentifier(
       UUID.fromString(splitSessionHandle(0)), UUID.fromString(splitSessionHandle(1)))
     val protocolVersion = TProtocolVersion.findByValue(splitSessionHandle(2).toInt)
-    val sessionHandle = new SessionHandle(handleIdentifier, protocolVersion)
-    backendService.closeSession(sessionHandle)
-    Response.ok().build()
+    new SessionHandle(handleIdentifier, protocolVersion)
```

Comment:
What if the sessionHandleStr is invalid

splitSessionHandle‘s size is 1?

or the generated sessionHandle is absent